### PR TITLE
Update AUTHORS and .mailmap

### PR DESCRIPTION
**Pull request by asmeurer**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -788,3 +788,19 @@ Ritu Raj Singh <RituRajSingh878@gmail.com>
 Rajiv Ranjan Singh <42106787+iamrajiv@users.noreply.github.com>
 Vera Lozhkina <veralozhkina@gmail.com>
 adhoc-king <46354827+adhoc-king@users.noreply.github.com>
+Mikel Rouco <mikel.mrm@gmail.com>
+Oscar Gustafsson <oscar.gustafsson@gmail.com>
+damianos <damianos@semmle.com>
+Supreet Agrawal <supreet11agrawal@gmail.com>
+shiksha11 <shiksharawat01@gmail.com>
```
Comment (asmeurer) [inline AUTHORS]: @shiksha11 if you want your name spelled out in the AUTHORS file, let me know.

File: AUTHORS [hunk 2]
```diff
@@ -788,3 +788,19 @@ Ritu Raj Singh <RituRajSingh878@gmail.com>
 Rajiv Ranjan Singh <42106787+iamrajiv@users.noreply.github.com>
 Vera Lozhkina <veralozhkina@gmail.com>
 adhoc-king <46354827+adhoc-king@users.noreply.github.com>
+Mikel Rouco <mikel.mrm@gmail.com>
+Oscar Gustafsson <oscar.gustafsson@gmail.com>
+damianos <damianos@semmle.com>
```
Comment (asmeurer) [inline AUTHORS]: @damianosSemmle if you want your name spelled out in the AUTHORS file let me know.

File: AUTHORS [hunk 3]
```diff
@@ -788,3 +788,19 @@ Ritu Raj Singh <RituRajSingh878@gmail.com>
...
+Supreet Agrawal <supreet11agrawal@gmail.com>
+shiksha11 <shiksharawat01@gmail.com>
+Martin Ueding <dev@martin-ueding.de>
+sharma-kunal <kunalsharma6914@gmail.com>
```
Comment (asmeurer) [inline AUTHORS]: @sharma-kunal if you want your name spelled out in the AUTHORS file, let me know.

**Comment by oscarbenjamin** [comment]
LGTM

File: .mailmap [hunk 4]
```diff
@@ -394,6 +394,10 @@ Gagandeep Singh <singh.23@iitj.ac.in> czgdp1807 <gdp.1807@gmail.com>
 Gagandeep Singh <singh.23@iitj.ac.in> czgdp1807 <36567889+czgdp1807@users.noreply.github.com>
 Gagandeep Singh <singh.23@iitj.ac.in> czgdp1807 <czgdp1807@gmail.com>
 Gagandeep Singh <singh.23@iitj.ac.in> MathLover <36567889+czgdp1807@users.noreply.github.com>
+Gagandeep Singh <singh.23@iitj.ac.in> czgdp1807 <singh.23@iitj.ac.in>
+Gagandeep Singh <singh.23@iitj.ac.in> Gagandeep Singh <36567889+czgdp1807@users.noreply.github.com>
+Gagandeep Singh <singh.23@iitj.ac.in> Gagandeep Singh <czgdp1807@gmail.com>
+Gagandeep Singh <singh.23@iitj.ac.in> Gagandeep Singh <singh.23@iitj.ac.in>
```
Comment (smichr) [inline .mailmap]: this appears to be a do-nothing line

Comment (smichr) [reply .mailmap]: not worth another test run, however

File: AUTHORS (hunk 2 above)
Comment (damianosSemmle) [reply AUTHORS]: @asmeurer Could you spell it as Damianos Christophides? Thanks!

Comment (asmeurer) [reply AUTHORS]: This was already merged. Can you send a PR fixing it. You need to edit `.mailmap` adding 

```
Damianos Christophides <damianos@semmle.com> damianos <damianos@semmle.com>
```

Then re-run `./bin/mailmap_update.py` to make sure it is correct, then `./bin/authors_update.py` to update AUTHORS, and commit both files.

**Pull request merged**